tests/nodeos_under_min_avail_ram.py

```diff
@@ -170,8 +170,8 @@ def setName(self, num):
                        Print("Failed to push create action to eosio contract for %d consecutive times, looks like nodeos already exited." % (timeOutCount))
                        keepProcessing=False
                        break
-                    Print("Failed to push create action to eosio contract. sleep for 60 seconds")
-                    time.sleep(60)
+                    Print("Failed to push create action to eosio contract. sleep for 5 seconds")
+                    time.sleep(5)
```

Comment:
Please keep track of test results for the next few days to make sure that if this case is hit that 5 seconds is enough time for all the operating systems.

Reply:
sure